ocs_ci/helpers/managed_services.py

```diff
@@ -456,6 +458,28 @@ def verify_faas_provider_resources():
     # Verify that the security groups are set up correctly
     assert verify_worker_nodes_security_groups()
 
+    # Check the presence of catalogsource and its state
+    catsrc = OCP(kind=constants.CATSRC, namespace=config.ENV_DATA["service_namespace"])
+    catsrc_info = catsrc.get().get("items")[0]
```

Comment:
Please try toe fetch the required catalogsource by name or other identifier. "Managed Fusion Agent" need not be the first item in the list. This works currenly becaue only one item will be present.

Reply:
Fixed